src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -1647,6 +1650,11 @@ neg-distrib-⊓-⊔ = antimono-≤-distrib-⊓ neg-mono-≤
 ∣p∣≃0⇒p≃0 {mkℚᵘ (ℤ.+ n)  d-1} p≃0ℚ = p≃0ℚ
 ∣p∣≃0⇒p≃0 {mkℚᵘ -[1+ n ] d-1} (*≡* ())
 
+0≤∣p∣ : ∀ p → 0ℚᵘ ≤ ∣ p ∣
+0≤∣p∣ (mkℚᵘ (ℤ.+ ℕ.zero) _)    = *≤* (ℤ.+≤+ ℕ.z≤n)
+0≤∣p∣ (mkℚᵘ (ℤ.+ (ℕ.suc n)) _) = *≤* (ℤ.+≤+ ℕ.z≤n)
```

Comment:
Can write `+0` instead `(ℤ.+ ℕ.zero)`, `+[1+ _ ]` instead of `(ℤ.+ (ℕ.suc n))` and `-[1+ _ ]` instead of `(ℤ.-[1+ n ])` to tidy things up a bit

Reply:
Fixed